eng/common/scripts/Prepare-Release.ps1

```diff
@@ -153,6 +142,17 @@ if ($LASTEXITCODE -ne 0) {
   exit 1
 }
 
+# Check API status if version is GA
+if (!$newVersionParsed.IsPrerelease)
+{
+  $isApproved = Check-ApiReviewStatus -PackageName $packageProperties.Name -packageVersion $newVersion -Language $LanguageDisplayName
+  if (!$isApproved)
+  {
+    Write-Error "API review is not approved in APIView system."
```

Comment:
I got feedback from the release drivers that we shouldn't fail here but instead we should make it a warning so that folks can still do the other prepare release stuff they need to while they get api approval in parallel. So lets turn this into a warning instead.

Reply:
done. Added as warning.